source/Grid/Grid.test.js

```diff
@@ -120,43 +120,49 @@ describe('Grid', () => {
   describe('shows and hides scrollbars based on rendered content', () => {
     const scrollbarSize = getScrollbarSize()
 
+    function renderTwice (markup) {
```

Comment:
Now in first render, overflowX and overflowY won't be set, so it can produce right result, whatever server-side rending or client-side rending.
And in follow up renders, because `this._scrollbarSize` was set, `overflowX` and `overflowY` will be set normally, to prevent #116 's problem